app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :recipes, dependent: :destroy
+
+  validates :name, presence: true
```

Comment:
- [ ] I think there might be a small issue in validating the user's name. As long as devise gem is installed, there is no need for validating its fields because it handles itself. Therefore your app is throwing me this error
![Screenshot (2)](https://user-images.githubusercontent.com/47625407/181804406-a104dd6b-5eed-48a8-9e75-1d992be0366f.png)